ledger-core/v2/deploy/kube/run.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+NUM_DISCOVERY_NODES=5
+INSOLAR_IMAGE="insolar/assured-ledger:latest"
+ARTIFACTS_DIR="/tmp/insolar"
+set -x
+check_dependencies() {
+  command -v docker >/dev/null 2>&1 || {
+    echo >&2 "docker is required. Aborting."
+    exit 1
+  }
+  command -v kubectl >/dev/null 2>&1 || {
+    echo >&2 "kubectl is required. Aborting."
+    exit 1
+  }
+  check_docker_images
+  check_ingress_installation
+}
+
+# Delete this after image templating will be done, and images will be in insolar hub
+check_docker_images() {
+  if [ "$(docker images $INSOLAR_IMAGE -q)" = "" ]; then
+    echo >&2 "make sure you made 'make docker_build'"
+    exit 1
+  fi
+}
+
+check_ingress_installation() {
+  if [ "$(kubectl get pods -n ingress-nginx -l app.kubernetes.io/name=ingress-nginx | grep -c Running)" = "0" ]; then
+    kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-0.31.0/deploy/static/provider/cloud/deploy.yaml
+  fi
+}
+
+delete_ingress() {
```

Comment:
also unused

Reply:
I don't know where to use it, but it well be usefull if you want to drop ingress that takes 80 port on your machine. I'll add this to readme

Reply:
Maybe in `stop.sh`?

Reply:
It takes some time when you're running locally, so i think we don't need to do it everytime